docs/caveats.md

```diff
@@ -78,16 +78,6 @@ It's important to select the correct context when you're debugging,
 otherwise you cannot access all the global variables,
 _save as temp variables_ also fails.
 
-#### "WebSocket connection to 'ws://localhost:8097/' failed"
```

Comment:
It seems like you're bringing irrelated changes into this PR.

Reply:
For resolving the performance issue,  you need me to rebase onto the master. When the issue is resolved,  the target branch will also rebase onto the master. These changes will disappear